### DISCUSSION TITLE
Bedrock Server: Attaching to ./bedrock_server process

### DISCUSSION THREAD

**Discussion by andrewkeithly** [itzg/docker-minecraft-bedrock-server#227] [Q&A]
First of all – thanks for creating this! What a cool way to learn about Azure and how to provision with Terraform.

I know Java Minecraft utilizes RCON to issue commands directly to server. With the container running the `./bedrock_server` on boot – I don't see a way to attach to the server process to issue any CLI. I suppose we could start the process in a tmux or screen? Not sure if you have any ideas on this. I tried killing the process so that I could manually start it attached but that immediately restarts the container (duh 🤦🏻‍♂️)

**Comment by andrewkeithly** [comment]
worth noting azure cli does provide a way to "attach" process via `az container attach` but this is just basically following the log. It doesn't allow you to issue commands like you might by attaching to a docker container

**Reply by itzg** [reply]
That's a bummer their `attach` doesn't attach in the same way as a local container. IMHO, seems redundant with the `logs` command.  There's a trick I found with the Java server container using a pipe, so I'll see if I can do similar with this one.

**Comment by itzg** [comment]
I have written an issue for the upstream enhancement I'll need to make: https://github.com/itzg/entrypoint-demoter/issues/8
